src/libraries/System.Security.Cryptography/ref/System.Security.Cryptography.cs

```diff
@@ -3131,6 +3202,7 @@ public sealed partial class X509SubjectKeyIdentifierExtension : System.Security.
         public X509SubjectKeyIdentifierExtension(System.Security.Cryptography.X509Certificates.PublicKey key, System.Security.Cryptography.X509Certificates.X509SubjectKeyIdentifierHashAlgorithm algorithm, bool critical) { }
         public X509SubjectKeyIdentifierExtension(string subjectKeyIdentifier, bool critical) { }
         public string? SubjectKeyIdentifier { get { throw null; } }
+        public System.ReadOnlyMemory<byte> SubjectKeyIdentifierBytes { get { throw null; } }
```

Comment:
The string property is nullable but this isn't?

Reply:
I made it nullable, and I think it made me sad.

The string is null if, and only if, you use the default constructor (and don't then do a CopyFrom).

If the extension was fed data then the property is non-null, or throws (because Decode failed).

So the question is: is it useful to distinguish "I'm an unpopulated SKId value" from "I represent the empty identifier"?  I'm not sure it is.

(The property could also just throw for the default ctor case... guidelines say to "AVOID" throwing, but since lazy Decode can anyways, and no one should really see it in this state... maybe throwing is better...)